src/main/resources/log4j.properties

```diff
@@ -0,0 +1,6 @@
+log4j.rootLogger=INFO, stdout
+
```

Comment:
Are we using log4j2? Why not use the log42j.xml file, then?

<img width="955" alt="CleanShot 2023-03-22 at 07 39 53@2x" src="https://user-images.githubusercontent.com/17779779/226812485-2b0abd90-6336-482b-ba08-475378e1f12c.png">